docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
     container_name: bunny_rabbitmq
     environment:
       RABBITMQ_NODENAME: bunny
+      BUNNY_RABBITMQ_HOSTNAME: mercurio
```

Comment:
Not sure what should be chosen as default. Please check my considerations in Google Groupe.
https://groups.google.com/d/msg/rabbitmq-users/V4fCdASYT1I/J5ss8yCzCwAJ